Update README.md (#1)
rxn4chemistry/rxnaamapper: README.md

## train the model

> wandb can be controlled with environment variables set before the script runs or within the script. 

Reviewer: ```suggestion
We use WandB for logging, if you don't have a mode configured you can simply disable it by setting:
```

> ```console

Reviewer: ```suggestion

```console
```

export WANDB_MODE=offline
```

> When you're running a script in an automated environment, you can control wandb with environment variables set before the script runs or within the script.

Reviewer: ```suggestion
Besides using an env variable, you can even create a custom script configuring the logging within it.
```

mapper = RXNAAMapper(config=config_mapper)
mapper.get_reactant_aa_sequence_attention_guided_maps(["NC(=O)c1ccc[n+]([C@@H]2O[C@H](COP(=O)(O)OP(=O)(O)OC[C@H]3O[C@@H](n4cnc5c(N)ncnc54)[C@H](O)[C@@H]3O)[C@@H](O)[C@H]2O)c1.O=C([O-])CC(C(=O)[O-])C(O)C(=O)[O-]|AGGVKTVTLIPGDGIGPEISAAVMKIFDAAKAPIQANVRPCVSIEGYKFNEMYLDTVCLNIETACFATIKCSDFTEEICREVAENCKDIK>>O=C([O-])CCC(=O)C(=O)[O-]"])
```

> Nb: The model path contains both the model and the config.json files. The files in this folder result from the model train turned into an HuggingFace model using the script from the previous section.

Reviewer: Replace `Nb:` with `*NOTE:*` :)

Maybe better:
The model path contains both the model binary file and the config.json. The files in this folder are generated from the model trained turned into an HuggingFace model using the script from the previous section.
